slapp/explorer/regions.py

```diff
@@ -163,3 +201,388 @@ def municipalities_details(ids: list[int]) -> list[Municipality]:
+                        "name": "Rüdersdorf bei Berlin",
+                        "x": 0,
+                        "y": 0,
+                        "itemStyle": {"color": "#798897"},
+                        "symbolSize": 30,
+                    },
+                    {
+                        "name": "Grünheide (Mark)",
+                        "x": 50,
+                        "y": 50,
+                        "itemStyle": {"color": "#798897"},
+                        "symbolSize": 30,
+                    },
+                    {"name": "Erkner", "x": -10, "y": 50, "itemStyle": {"color": "#798897"}, "symbolSize": 30},
+                    {"name": "Netz", "x": 80, "y": -25, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                    {"name": "Netz ", "x": 60, "y": 75, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                    {"name": "Netz  ", "x": -20, "y": 75, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                    {"name": "Netz   ", "x": -20, "y": -20, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                ],
+            },
+        ]
+    elif region == "Kiel":
+        energy_data = [
+            {
+                "title": "Stromaustausch (Kiel)",
+                "energyData": [
+                    {"source": "Kiel", "target": "Netz", "value": 120},
+                    {"source": "Netz", "target": "Kiel", "value": 20},
+                ],
+                "nodes": [
+                    {"name": "Kiel", "x": 0, "y": 0, "itemStyle": {"color": "#798897"}, "symbolSize": 30},
+                    {"name": "Netz", "x": 25, "y": 0, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                ],
+            },
+            {
+                "title": "Stromaustausch (Kiel)",
+                "energyData": [
+                    {"source": "Kiel", "target": "Netz", "value": 120},
+                    {"source": "Netz", "target": "Kiel", "value": 20},
+                ],
+                "nodes": [
+                    {"name": "Kiel", "x": 0, "y": 0, "itemStyle": {"color": "#798897"}, "symbolSize": 30},
+                    {"name": "Netz", "x": 25, "y": 0, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                ],
+            },
+        ]
+    return energy_data
+
+
+def flow_chart(request: HttpRequest) -> JsonResponse:
+    """Return requested data."""
+    chart_type = request.GET.get("type", "verbu")
+
+    if chart_type == "verbu":
+        flow_data = get_energy_data("verbu")
+    elif chart_type == "einzeln":
+        flow_data = get_energy_data("einzeln")
+    elif chart_type == "kiel":
+        flow_data = get_energy_data("Kiel")
+    else:
+        flow_data = {
+            "title": "Unbekannter Typ",
+            "energyData": [],
+            "nodes": [],
+        }
+
+    response_data = {"data": flow_data}
+
+    return JsonResponse(response_data)
+
+
+def cost_capacity_chart(request: HttpRequest) -> JsonResponse:
```

Comment:
Same here - please move view to views.py and calculation to regions.py in order to separate logics